plugin/storage/cassandra/schema/migration/v001tov002.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+function usage {
+    >&2 echo "Error: $1"
+    >&2 echo ""
+    >&2 echo "Usage: KEYSPACE={keyspace} $0"
+    >&2 echo ""
+    >&2 echo "The following parameters can be set via environment:"
+    >&2 echo "  KEYSPACE           - keyspace "
+    >&2 echo ""
+    exit 1
+}
+
+confirm() {
+    read -r -p "${1:-Are you sure? [y/N]} " response
+    case "$response" in
+        [yY][eE][sS]|[yY])
```

Comment:
You can probably just do `if [[ "$(echo $response | tr 'a-z' 'A-Z')" =~ "Y(ES)?"]]`.

Reply:
I don't think that would work

Reply:
I'm not saying it's necessarily easier, but why wouldn't it work?

Reply:
I don't know, I didn't spend time debugging it
```
bash-3.2$ response=Y && if [[ "$(echo $response | tr 'a-z' 'A-Z')" =~ "Y(ES)?" ]]; then echo "passed"; fi
bash-3.2$
```

Reply:
Right. I think bash regex might just be awful.